slaves/linux-cross/arm-linux-gnueabihf.config

```diff
@@ -0,0 +1,577 @@
+#
+# Automatically generated file; DO NOT EDIT.
+# Crosstool-NG Configuration
+#
+# Generated with crosstool-ng-1.22.0's `ct-ng menuconfig` command using the default configuration
+# plus the following modifications
+#
+# Target options > Target Architecture = arm
+# Target options > Architecture level = armv7-a -- (*)
```

Comment:
Yeah we can certainly punt this to later, the only real issue here I believe is that `arm-unknown-linux-gnueabihf` is armv6 by default for us instead of armv7. I *think* that exporting `CC_armv7_unknown_linux_gnu` will be enough to hijack the entire rustbuild build system, but I haven't tested that yet.